content/index.fr.md

```diff
@@ -0,0 +1,230 @@
+---
+# Translation info https://www.w3.org/wiki/WAI/Website/Translate
+
+title: "Vue d'ensemble de l'évaluation de l'accessibilité Web"
+nav_title: "Vue d'ensemble : Evaluation"
+
+lang: fr
+last_updated: 2020-12-27
+permalink: /test-evaluate/fr
+
+translators: 
+- name: "Sofia Ahmed"
+contributors:
+- name: "Sandra Velarde Gonzalez (ETNIC)"
+github:
+  repository: w3c/wai-eval-overview
+  path: content/index.fr.md
+
+feedbackmail: wai@w3.org
+class: tight-page
+footer: >
+  <p>Note à propos de l'audiodescription : les vidéos présentes sur cette page n'incluent pas d'audiodescription synchronisée car les images n'illustrent que l'audio et ne fournissent pas d'informations supplémentaires. Dans ce cas-ci, l'audiodescription serait plus gênante qu'utile pour la plupart des utilisateurs, y compris les personnes qui ne peuvent pas voir les images. La description des informations contenues dans les images est reprise dans la transcription avec la description des images ("transcription descritive").</p>
+  <p><strong>Date :</strong> Mise à jour : 28 avril 2020.</p>
```

Comment:
I could be wrong but per the review instruction, the date in the footer must be matching one of the dates at the beginning of the page (about the traduction). There are 3 dates on the page, and they are all different.